pjlib/include/pj/config.h

```diff
@@ -1415,7 +1415,7 @@ PJ_BEGIN_DECL
  * Extra suffix for the version (e.g. "-trunk"), or empty for
  * web release version.
  */
-#define PJ_VERSION_NUM_EXTRA	"-12"
+#define PJ_VERSION_NUM_EXTRA	"-2"
```

Comment:
Why version extra is changed from 12 to 2?

Reply:
because it should be 2.13.1-2 and not 2.13.1-12, there was only one release with 2.13.1 changes

Reply:
thx!